apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/api/common/cql/builder/QueryBuilderTest.java

```diff
@@ -335,6 +335,9 @@ public static Arguments[] sampleQueries() {
       arguments(
           new QueryBuilder().select().count("a").from("ks", "tbl").build().getCql(),
           "SELECT COUNT(a) FROM ks.tbl"),
+      arguments(
+          new QueryBuilder().select().count().from("ks", "tbl").build().getCql(),
+          "SELECT COUNT(1) FROM ks.tbl"),
```

Comment:
What about `COUNT (*)`? Should we test that as well?

Reply:
@jeffreyscarpenter Both count(1) and count(*) works the same way, so in the QueryBuilder when count() is used I have to pick one way so  used count(1).